modules/tensor_mechanics/src/materials/ComputeStressBase.C

```diff
@@ -25,8 +25,10 @@ ComputeStressBase::ComputeStressBase(const InputParameters & parameters) :
     _stress(declareProperty<RankTwoTensor>(_base_name + "stress")),
     _elastic_strain(declareProperty<RankTwoTensor>(_base_name + "elastic_strain")),
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
+    _dstrain_dT(getDefaultMaterialProperty<RankTwoTensor>(_base_name + "dstrain_dT")),
     _extra_stress(getDefaultMaterialProperty<RankTwoTensor>(_base_name + "extra_stress")),
     _Jacobian_mult(declareProperty<RankFourTensor>(_base_name + "Jacobian_mult")),
+    _dstress_dT(declareProperty<RankTwoTensor>(_base_name + "dstress_dT")),
```

Comment:
Please use the DerivativeMaterial interface for this. We should not have any hardcoded _derivative_ property names in tensor mechanics...

...But this class doesn't even couple to the temperature variable...

...so this class does not seem like the right place to declare this property at all.